l10nfr.properties

```diff
@@ -19,3 +19,9 @@ defaults.mavenArtifactId=sonar-l10n-fr-plugin
 9.8.0.date=2023-01-04
```

Comment:
Only one version at a time can be marked compatible with `LATEST`. When you archive a version, you should update its compatibility range to end with the currently released version so everyone who's already installed the old version will be prompted to upgrade.